CorsixTH/Lua/languages/english.lua

```diff
@@ -255,6 +257,7 @@ tooltip.main_menu = {
   options = "Tweak various settings",
   exit = "No, no, please don't leave!",
   quit = "You are about to quit from CorsixTH.   Are you sure this is what you want to do?",
+  connect_debugger = "Connect CorsixTH to a DBGp Lua debugger server. There's a CorsixTH Wiki debugger tutorial. All CorsixTH programmers should use a Lua debugger to help them quickly & easily locate bugs."
```

Comment:
I find that last sentence patronizing.  A link to the debugger tutorial would be appreciated though.